src/sql/src/plan/statement/ddl.rs

```diff
@@ -715,9 +715,14 @@ pub fn plan_create_source(
             namespace,
             table,
             columns,
+            slot,
         } => {
             scx.require_experimental_mode("Postgres Sources")?;
 
+            let slot_name = slot
```

Comment:
Nit: maybe `Postgres sources must provide a slot` (to avoid including the `plan` bit)

Reply:
Will fix

Reply:
fixed!